pages/learn/getting-started.md

```diff
@@ -1,85 +1,68 @@
 
 <img alt="Network configuration" src="/img/common/app/network.png" width="80%">
 
-Clicking *Advanced* presents the option to select the rate at which your device checks for updates:
-
-<img alt="Advanced options" src="/img/common/app/advanced.png" width="80%">
+Select the type of network connection you'll be using: *Ethernet Only* or *Wifi + Ethernet*. A network connection is required to allow the device to connect to {{ $names.cloud.lower }}. Selecting *Wifi + Ethernet* allows you to enter a *Wifi SSID* and *Wifi Passphrase* which is then built into the image.
 
-Once you have finished your image configuration, click the *Download {{ $names.os.lower }}* button. When the download completes, you should have a zipped image file with a name like `{{ $names.company.short }}-First-Fleet-2.47.1+rev1-v10.6.27.img.zip`, where `First-Fleet` is the name you gave your fleet on the dashboard.
-Alternatively, by clicking on the toggle section of the download button, you also have the option to download just a configuration file (`config.json`) rather than an entire image.
-
-<img alt="Advanced options" src="/img/common/app/download_image_or_config.png" width="80%">
+<img alt="Advanced options" src="/img/common/app/download_image.png" width="50%">
 
+Finally, click the **Download {{ $names.os.lower }}** button. When the download completes, you should have a zipped image file with a name like `{{ $names.company.short }}-First-Fleet-{{ $device.id }}-2.80.3+rev1-v12.7.0.img.zip`.
 
 ## Provision device
 
 <!-- This section is heavily customized based on the device -->
 {{import "getting-started/getDeviceOnDash"}}
 
-## Add release
+**Troubleshooting:** If you're not able to get the device to appear in the dashboard, take a look at [our troubleshooting guide][troubleshooting] or try our [support channels][support].
+
+## Create a release
 
-Now that we have a device or two connected to a {{ $names.company.lower }} fleet, let's deploy some code.
+Now that a device is online in your fleet, you can push code to it, which creates a release. This release will then be applied to all the devices present in your fleet.
 
-The recommended way to deploy code is to use the {{ $names.cli.lower }}. Follow the [installation
-instructions][cli-install]. After it is installed, login to your {{ $names.company.lower}} account
+The recommended way to deploy code is to use the [{{ $names.cli.lower }}](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md) as mentioned in the requirements above. After it is [installed][cli-install], login to your {{ $names.company.lower}} account
 with the `{{ $names.company.lower }} login` command on the terminal:
```

Comment:
Only a semantic thing, but I wonder if `{{ $names.company.lower }} login` should be e.g. `{{ $names.cli_command.lower }} login` ? (just in case the company-name is ever different to the cli-command? :man_shrugging: )

Reply:
> just in case the company-name is ever different to the cli-command?

@lurch Indeed, I have thought about it. Just for discussion's sake, I consider our CLI to not be a product rather an interface to interact with the product (balenaCloud) just like the API or the SDK. So, I don't see that possibility ever occurring where the company name might be different than CLI command. Seems weird in my head. 

Example: AWS changing their name to SWA. Then, still calling it [AWScli](https://aws.amazon.com/cli/) would be quite weird.